src/pps/pps.pb.go

```diff
@@ -120,9 +120,9 @@ type JobInfo struct {
 	Pipeline     *Pipeline                   `protobuf:"bytes,3,opt,name=pipeline" json:"pipeline,omitempty"`
 	Shards       uint64                      `protobuf:"varint,4,opt,name=shards" json:"shards,omitempty"`
 	Inputs       []*JobInput                 `protobuf:"bytes,5,rep,name=inputs" json:"inputs,omitempty"`
-	ParentJob    *Job                        `protobuf:"bytes,6,opt,name=parent_job" json:"parent_job,omitempty"`
-	CreatedAt    *google_protobuf1.Timestamp `protobuf:"bytes,7,opt,name=created_at" json:"created_at,omitempty"`
-	OutputCommit *pfs.Commit                 `protobuf:"bytes,8,opt,name=output_commit" json:"output_commit,omitempty"`
+	ParentJob    *Job                        `protobuf:"bytes,6,opt,name=parent_job,json=parentJob" json:"parent_job,omitempty"`
```

Comment:
This part seems a little weird, I'm not quite sure what happens when you have 2 json tags for the same field. It might break some of the indexes we have in Rethink (not on these protos but on the ones in `src/pps/persist/persist.pb.go`).